### ISSUE TITLE
EA Team Planning Agenda

### ISSUE THREAD

**Issue by jamiesiu** [hackforla/expunge-assist#670]
### Overview
This issue tracks agenda for Monthly Meeting for All Teams. The meeting is the first (full) week of every month on Friday at 10:30 AM PT. Product Roadmap in Miro Board is attached in #615 

# [YR-MO-DAY] Agenda and Notes

### Roll Call
- [ ] Product Team
- [ ] UXR team
- [ ] Content team
- [ ] Desigm team 
- [ ] Development team 

### What's Next Month / What we need from each other / Dependencies (by team)

Resources: Once new meeting time confirmed, update to look more like https://github.com/hackforla/internship/issues/461

**Comment by ValeriyaMetla** [comment]
## 9/9 Meeting Notes

### Review of Past Month
#### Design 
* responsive designs
* want to complete LP changes / short input boxes - carrying over to this month
* advice page for LG 
* Footer mobile update 
#### Content 
* must-haves 
* Irene got a job and transitioned from project, Sam main person for now 
* user journey tabled for later 
* source of truth for Content (Figma) - lot of work
* Content Design System deferred to after IUT2
#### Research
* onboarded 3-4 new ppl
* develop basic user journey 
* IUT2 - research plan and task plan in place from IUT1. want to know more about design changes 
* outreach plan good to go. Survey planning to send to legal clinics is ready. Survey for target users for insight into terminology 
#### Dev 
* few changes from IUT1 implemented
* slowing down bc other devs Sydney busy with work and Daniel transitioning out likely soon. Will need to find / recruit new devs.
* want to get to MVP state asap so just bug fixes later. 

### Our Overall Direction 
Collab does take time. Some scope added 
* Content has 4 issues to complete 
* What's possible in Dev? new functionality or re-organizational asks? LG categories addition. Shifting naming, option to add more children, variable sentences. Changes seem reasonable to Dev. 
* Design changes - finalizing categories view, short form length, LP reorg - aiming to show by next Dev Design meeting. Past dependencies - Design needs to update certain aspect e.g. Content width not consistent or unclear. Better now since we have this info
* we agree to move forward with IUT2 efforts and also pursue survey outreach in parallel

### What's Next Month / What we need from each other / Dependencies 
What do we need to get this in front of users? Can we just test with real users now? 

#### Dev 
* #659 - this is ready for dev
* assign tickets directly to Daniel for now since it's mainly them for Dev

#### Content
* terminology survey in collab with UXR
* which options to go forward with 
* go back to Content Design System work once must-haves done
* Sam sharing what's happening on Content/Design over to UXR

#### Research
* surveys ready to go in few weeks. need Bonnie approval prior to distribution
* planning for IUT2. need from Design/Content to know about scope and what feedback they need 
* Partner Outreach Survey goal - get a sense of how legal clinics use letters, establish prospects/potential partners
* UXR - what do we want to know from must-have feedback (Design/Content changes) 

#### Design
* get feedback for wireframes e.g. parenting flow, LP potential updates
* get FAQ page done, update popups (see what's possible/not in coming weeks) 
* Splitting up work so can get to Dev quicker - illustrations, parenting flow, etc

#### Product
* collective support for teams
* checking with Bonnie
* Public facing wiki - not urgent? we need this before putting in front of target users (lower priority than and not blocker for IUT2 and survey outreach). Consent for tester users. Need for flexibility for last minute changes. Options like Notion?


###

**Comment by ValeriyaMetla** [comment]
@jamiesiu can you schedule in Google calendar this meeting till the end of the year, for October, November and December. We can identify exact dates on Monday 9/19

**Comment by jamesshin27** [comment]
### 2/3 Meeting Notes

Attendees:
Anita Morales
Kelsey Robbins
Serena Sanders
Sheron Virani
Sydney Walcoff
Thomas Demoner
Yumeng Luo
Sofia Laguna
Kevin Moutoucarpin

**Review of Past Month (by team)**
Design
- Assessed team's bandwidth and improving team's workflow
- Focusing on design system, collaborating with the content team

Research
- Working on research documentation (past and current)
- Team has gotten drastically smaller, onboarding new team member (Sofia is new)
- Brainstormed research objectives for the roadmap and finishing planning the objectives for February
- Handed over IUT2 plan from content/design team and will be creating research plan (possibly just for the landing page for now? waiting for the dev team)
- Kelsey recorded a github tutorial (in the Google drive research team): https://drive.google.com/drive/u/0/folders/1IGHX-b8aDYGh1zRsYuV_XpHiFFjzbkyh

Dev
- Recruiting and interviewing new developers
- Working on current issues 
- Can't onboard Phillip to have him start working on the project
- Landing page still needs to be updated from design for the research team to start user testing
- Design can create a protoype on Figma for user testing if the dev team is too busy but Dev will prioritize the landing page: 

**Our Overall Direction**

Design
- Making sure all the wireframes are updated (mobile and web), working on it in Feb
- Auditing the wireframes in February to create a standard guideline
- Working on documentation and having all the information in one place for transparency: Archiving and saving older designs
- Outlining the start page for the dev team

Research
- Recruiting more researchers in February
- Adding research into table of contents to have more transparency during onboarding

Dev
- Updating documentation for onboarding
- Prioritizing the landing page

**What's Next Month / What we need from each other / Dependencies (by team)**
- Talk about AI (chatGPT) and how that impacts our team in our all team meeting next week
- How our website protects privacy of the people compared to AI
- **Dev** team will prioritize making updates to the landing page based on the design team's feedback so that the research team can start user testing.

**Comment by jccchurch** [comment]
# [5/5/23] Agenda and Notes

### Roll Call
- [x] Product Team
- [x] UXR team
- [x] Content team
- [x] Design team 
- [x] Development team 

### Attendees
Product team:
UXR team: 
Content team:
Design team: 
Development team: 

### Miro board 
Access is updated. Current link: https://miro.com/welcomeonboard/Z0tpZWFDZ3JqR3k2YUxoak9PSk5HdGNzMEZmMTAzVlgyM0JpQk9aSFVOV3lCakJhOHFGVDhUSzFjQU44SHllaXwzNDU4NzY0NTE4MjczODQ4ODg0fDI=?share_link_id=684428338363 

### Review of Past Month & Upcoming Direction

**Product**

- New PMs - Analicia & Lane
- Setting new PM-ORG meeting w/ Bonnie

**Design**

- New Designers - Michelle & Liz
- Audit color styles and standardize colors
- Clean UX Design related issues on board
- Finalize Figma mobile wireframes/wrapping up IUT2 - shared w research team - scheduling a time

**Content**

- Figma & Content SOT 
- Clean UX Design related issues on board
- Content guidelines - Continue
- Finalize Figma mobile wireframes/wrapping up IUT2
- Rework Product Principles cross functionally

**Research**

- Continue to distribute surveys - main priority
- Complete IUT2 - other priority - need more people recruited within HFLA
- Complete survey analysis - trying to finish 
- Recruit new co-lead

**Dev**

- New PM - Lane
- A couple new members dropped 
- Finished issues for IUT2?
- Need to clean up open issues on board

### What's Next Month / What we need from each other / Dependencies

- Project Goals, Principles & Product Vision
- Reassessing the Roadmap & Milestones (official launch for web & mobile?)
- Update One Pager: https://docs.google.com/document/d/1JFZGKxeEBWVak_GATgIt0Knyr9_gJ15CBirynebb4tg/edit 
- Project Branding: Zoom Background, About section on landing page, No longer under Github, etc
- Collaboration with a legal clinic/Grammarly/language assistance?
       - What's useful for letter generators? How can we target it to the judge/lawyer audience through storytelling? 
- Open to team



**- NEW TEAM MEMBERS/ TEAM NEEDS** 

- UXR is on an island - arrange a meeting once/twice a month for Des/Content/UXR collab
- Onboarding process improvement: https://github.com/hackforla/expunge-assist/issues/666 
- Team meeting times - opening up meetings to encourage cross-functional collaboration

**Comment by amejiamesinas** [comment]
# [6/2/23] Agenda and Notes

### Roll Call
- [ ] Product Team
- [ ] UXR team
- [ ] Content team
- [ ] Design team 
- [ ] Development team 

### Attendees
Product team:
UXR team: 
Content team:
Design team:
Development team: 

### Miro board 
Access is updated. Current link: https://miro.com/welcomeonboard/Z0tpZWFDZ3JqR3k2YUxoak9PSk5HdGNzMEZmMTAzVlgyM0JpQk9aSFVOV3lCakJhOHFGVDhUSzFjQU44SHllaXwzNDU4NzY0NTE4MjczODQ4ODg0fDI=?share_link_id=684428338363 

### Meeting Goals 
(From Miro Board)
-  Product Demo for Bonnie
-  Define our user stories
- QUESTIONS:
1. What is the teams biggest road block at the moment?
2. What needs to be accomplished before a product demo?
3. How long do you estimate that will take?

**Product**

**Design**

**Content**

**Research**

**Dev**


### What's Next Month / What we need from each other / Dependencies

- Project Goals, Principles & Product Vision
- Reassessing the Roadmap & Milestones (official launch for web & mobile?)
- Update One Pager: https://docs.google.com/document/d/1JFZGKxeEBWVak_GATgIt0Knyr9_gJ15CBirynebb4tg/edit 
- Project Branding: Zoom Background, About section on landing page, No longer under Github, etc
- Collaboration with a legal clinic/Grammarly/language assistance?
       - What's useful for letter generators? How can we target it to the judge/lawyer audience through storytelling? 
- Open to team



**- NEW TEAM MEMBERS/ TEAM NEEDS** 

- UXR is on an island - arrange a meeting once/twice a month for Des/Content/UXR collab
- Onboarding process improvement: https://github.com/hackforla/expunge-assist/issues/666 
- Team meeting times - opening up meetings to encourage cross-functional collaboration


### Meeting Notes

**Content**


**Research**

**Team Needs**



### Goals for Next Month


###  **Open Action Questions/Items:** 

- How can the Research team's work help the other teams?
- Who is our User? - What organizations/people are interested in the EA project? Who are we helping? Who is getting the letter and is it in the appropriate language?  
- What is our overall Product Vision? How will this affect our branding/content/language/research/etc
- Who is our marketing/social/Portfolio Manager person or team - can we establish one?
- Examine & Redefine the Product Roadmap based on these factors 
- Meet with Bonnie to align goals and ideas (meeting time TBD)
- Need more info on MERL transition & correct language 
- ORG chart & Team meetings - Miro board for a visual - how can we utilize without creating extra work/things to update?

**Comment by amejiamesinas** [comment]
# [7/7/23] Agenda and Notes

### Roll Call and Introductions 
- [ ] UXR team
- [ ] Content team
- [ ] Design team 
- [ ] Development team 

### Attendees
UXR team: 
Content team: 
Design team: 
Development team: 

### Meeting Goals 
Brainstorm EA attributes and begin drafting Mission Statement 
Use [Figma Board](https://www.figma.com/file/Crqy17T5GSG4cbRxF1XamO/EA-COMP-ANALYSIS?type=whiteboard&node-id=580-557&t=8XLMTd9sMeXSgdh1-4) to brainstorm Mission Statement. 

Additional Updates
- Bonnie will demo product on 7/7
- PM team has follow up discussion with Bonnie on 7/21
- Meetings will guide ongoing vision planning and roadmapping for team 


### Meeting Notes

### Goals for Next Month
- Complete Product Demo and revise roadmapping and planning 

###  **Open Action Questions/Items:**

**Comment by lmitch10** [comment]
##Roll Call and Introductions
•	 UXR team
•	 Content team
•	 Design team
•	 Development team
##Attendees
UXR team: Analicia, Danielle, Rong Ann, Beth, Genggeng, Travis, 
Content team: Sam, 
Design team: Anita, 
Development team: Sydney, Lane
Meeting Goals
Brainstorm EA attributes and begin drafting Mission Statement
Use [Figma Board](https://www.figma.com/file/Crqy17T5GSG4cbRxF1XamO/EA-COMP-ANALYSIS?type=whiteboard&node-id=580-557&t=8XLMTd9sMeXSgdh1-4) to brainstorm Mission Statement.
##Additional Updates
•	Bonnie will demo product on 7/7
•	PM team has follow up discussion with Bonnie on 7/21
•	Meetings will guide ongoing vision planning and roadmapping for team
##Meeting Notes
-	See Figma Notes
##Brainstorming Needs:

- Process we use- how can improve process around Github to ensure easily flow of communication between teams 
- what is the hope/plan for how people (legal aid, lawyers, people who want to expunge) will find out about Expunge Assist?
- How to make use of this activity, and what is our process from this kind of activity?
##Goals for Next Month
•	Complete Product Demo and revise roadmapping and planning
##Open Action Questions/Items:

**Comment by amejiamesinas** [comment]
# [9/1/23] Agenda and Notes

### Roll Call and Introductions 
- [ ] UXR team
- [ ] Content team
- [ ] Design team 
- [ ] Development team 

### Attendees
UXR team: 
Content team: 
Design team: 
Development team: 

### Agenda
[ ] EA Team Updates
[ ] EA Team Needs
[ ] IUT 2 Insights 
[ ] Next Steps

### Meeting Notes

### Goals for Next Month


###  **Open Action Questions/Items:**

**Comment by lmitch10** [comment]
# [9/1/23] Agenda and Notes

### Roll Call and Introductions 
- [ ] UXR team
- [ ] Content team
- [ ] Design team 
- [x] Development team 

### Attendees
UXR team: 
Content team: 
Design team: 
Development team: 

### Agenda
[ ] EA Team Updates
[ ] EA Team Needs
[ ] IUT 2 Insights 
[ ] Next Steps

### Meeting Notes
Roadmap changes
Julia stepping down, Lane and Analicia will be vetting the potential new PMs

Content: 
Voice and tone updates
New words/keywords
Update content guidelines
Prioritization after IUT2
Dev: 
Onboarding
Behind the scenes tickets fulfilled
Waiting for new tickets
Project Manager will be hopping on meetings next week

UIT2: 
Sample base had UX experience
Sample base had never used the EA page 
Easy navigation and easy to understand how to use the product
Potential “how it works” to make content more accessible
Trust/Data Security
Need to match the tone

**Comment by amejiamesinas** [comment]
# [10/06/23] Agenda and Notes

### Roll Call and Introductions 
- [ ] UXR team
- [ ] Content team
- [ ] Design team 
- [ ] Development team 

### Attendees
UXR team: 
Content team: 
Design team: 
Development team: 

### Agenda
[ ] Notetaker?
[ ] EA Team Updates
- [ ] Team Needs
- [ ] Onboarding
- [ ] Offboarding
- [ ] Updating open roles for each team 
- [ ] Slack Channel Bookmarks

[ ] Issues, timeline & roadmapping for each team
- [ ] Milestone: Prep Launch
- [ ] Identifying priorities for launch prep with timeline
- [ ] Workflow clarification: Separate or Joint Issues? 
- [ ] Next Steps for Accessibility? 


### Meeting Notes

**Comment by amejiamesinas** [comment]
# [11/03/23] Agenda and Notes

### Roll Call and Introductions 
- [ ] UXR team
- [ ] Content team
- [ ] Design team 
- [ ] Development team 

### Attendees
UXR team: 
Content team: 
Design team: 
Development team: 

### Agenda
[ ] Notetaker?
[ ] EA Team Updates
- [ ] Teams are growing, welcome new members
- [ ] Open positions: Design PM, and Co-lead for Research Team
- [ ] December Update: Hack for LA closed for December, no meetings
- [ ] Hack for LA Events in December: See Bonnie post in [Slack](https://hackforla.slack.com/archives/C04502L0P/p1698192892460479) 
- [ ] New Team Roster](https://github.com/hackforla/expunge-assist/wiki/The-Current-Team)
- Autofilled from Roster
- Please edit as needed
- [ ] [Share/ Review Project and Team Roadmapping](https://www.figma.com/file/SRsM0BvWhtmFwpg23bKTLB/EA-Collaboration-(Design%2FContent)?type=whiteboard&node-id=3152-10049&t=WnuLglsKFNivzuQV-4)
- [ ] [End of Year Exercise](https://www.figma.com/file/SRsM0BvWhtmFwpg23bKTLB/EA-Collaboration-(Design%2FContent)?type=whiteboard&node-id=3303-9784&t=hKYw6u5f3TJHdFU4-4)




### Meeting Notes

**Comment by amejiamesinas** [comment]
# [02/02/24] Agenda and Notes

### Roll Call and Introductions 
- [ ] UXR team
- [ ] Content team
- [ ] Design team 
- [ ] Development team 

### Attendees
UXR team: 
Content team: 
Design team: 
Development team: 

### Agenda
- [ ] Notetaker:
- [ ] Product Updates
- [ ] Design Updates
- [ ] Content Updates
- [ ] Research Updates
- [ ] Dev Updates
- [ ] Roadmapping and Priority Issues
- [ ] Any other updates?

### Meeting Notes

**Comment by awellsbiz** [comment]
**### Notes for 2/2/24 ALL TEAM MEETING**

**Attendance / Roll Call / Intros:**

- [ ] Analicia
- [ ] Sydney
- [ ] Anthony
- [ ] David
- [ ] Mireya
- [ ] Sara
- [ ] Sylvia
- [ ] Emma
- [ ] Justin
- [ ] Anita
- [ ] Rong
- [ ] Cheryl
- [ ] Sam

- Curious to see what other teams are up to? Feel free to join in on a weekly meeting!

**Product Updates:**

- Prepping for Launch

**Design Updates:**

- Recap: Design systems
- Plan for Feb: Credits page, About us page, finalizing typography
- No need for more designers. If need more Analicia is there to help

**Content Updates:**

- Tag all members when messaging
- waiting on Research for prep launch
- credits pages and accessibility
- Voice and Tone 

**Research Updates:**

- Reorienting priorities due to capacity
- Been figuring out "The best" terminology.
- User journey: trying to touch base with legal professionals for research purposes. 

**Dev Updates:**

- Documentation
- Looking forward to Accessibility

Roadmapping by team:

- Design:  PM to meet with Amanda (Branding) to create issues for branding in order to track progress. 
- Content: refining voice and tone, credits page, accessibility, and review roadmap
- Product:
- Research: Terminology Survey,  Revisiting User Journey, 

Overall Roadmap:

- Launch Prep: How are we gauging when we are done?
-  Bonnies Requirements and Accessibility 
- Edit roadmap to where we are right now
- Add post clean up to roadmap 
- We are in Launch Prep - fall under one umbrella for every team - no more milestones. 
- Make a issue for warning users about beta version of product. 

Other Updates:

**Comment by amejiamesinas** [comment]
# [03/01/24] Agenda and Notes

### Roll Call and Introductions 
- [ ] UXR team
- [ ] Content team
- [ ] Design team 
- [ ] Development team 

### Attendees
UXR team: 
Content team: 
Design team: 
Development team: 

### Agenda
- [ ] Notetaker:
- [ ] Documentation Process
- [ ] Collaboration
- [ ] One Pager
- [ ] Roadmap


### Meeting Notes

**Comment by amejiamesinas** [comment]
# [04/05/24] Agenda and Notes

### Roll Call and Introductions 
- [ ] UXR team
- [ ] Content team
- [ ] Design team 
- [ ] Development team 

### Attendees
UXR team: 
Content team: 
Design team: J
Development team: 

### Agenda
- [ ] Notetaker: 
- [ ] Current Milestone Goals
     - Address Bonnie identified issues
     - Preparing to Submit to Hacker 1 (https://www.hackerone.com/company/open-source-community)
     - Accessibility (part 1)
     - Can Also Reference: [FigJam](https://www.figma.com/file/SRsM0BvWhtmFwpg23bKTLB/EA-Collaboration-(Design%2FContent)?type=whiteboard&node-id=3738-9686&t=owon0cdOIlHvUap2-4)
- [ ] Team Updates
    - Setting Milestone [Launch Prep](https://github.com/hackforla/expunge-assist/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22Launch+Prep%22) Target Dates
    - [Dev Team](https://github.com/hackforla/expunge-assist/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22Launch+Prep%22+label%3A%22role%3A+development%22)
    - [Research Team](https://github.com/hackforla/expunge-assist/issues?q=is%3Aopen+is%3Aissue+label%3A%22role%3A+research%22+milestone%3A%22Launch+Prep+2.0%22+)
    - [Content Team Team](https://github.com/hackforla/expunge-assist/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22Launch+Prep%22+label%3A%22role%3A+UX+content+writing%22)
    - D[esign Team](https://github.com/hackforla/expunge-assist/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22Launch+Prep%22+label%3A%22role%3A+design%22)
- [ ] Research Presentation on color and Typography


### Meeting Notes

**Comment by amejiamesinas** [comment]
### Overview
This issue tracks agenda for Monthly Planning Meeting for PMs and Leads. The meeting is the first (full) week of every month on Friday at 10.30 AM PST. Product Roadmap in Miro Board is attached in #615 

# [Date] Agenda and Notes

### Roll Call
- [ ] Product Team
- [ ] UXR team
- [ ] Content team
- [ ] Desing team 
- [ ] Development team 

### Team Updates 
- [ ] Product Team
- [ ] UXR team
- [ ] Content team
- [ ] Desing team 
- [ ] Development team

### Stakeholder Review 
- [ ] #1410 

### What's Next Month / What we need from each other / Dependencies (by team)

**Comment by sylvia-nam** [comment]
# [2024-09-06] Agenda and Notes

### Roll Call
- [ ] Product Team
- [ ] UXR team
- [ ] Content team
- [ ] Design team 
- [ ] Development team 

### Team Updates 
- [ ] Product Team
- [ ] UXR team
- [ ] Content team
- [ ] Design team 
- [ ] Development team

### Stakeholder Review 
- [ ] #1410 

### Notes
### What's Next Month / What we need from each other / Dependencies (by team)

**Comment by vanessaavviles** [comment]
# [2024-10-03] Agenda and Notes

### Roll Call
- [ ] Product Team
- [ ] UXR team
- [ ] Content team
- [ ] Design team 
- [ ] Development team 

### Review of Past Month (by team)

### Our Overall Direction

### What's Next Month / What we need from each other / Dependencies (by team)

**Comment by mariaweissman** [comment]
Closing this as a historical agenda, as we are now using https://github.com/hackforla/expunge-assist/issues/1620

**Comment by raeraeicky** [comment]
# [2025-01-31] Agenda and Notes

### Meeting Facilitator:
### Meeting Notetaker:

### Roll Call
- [ ] Product Team
- [ ] UXR team
- [ ] Content team
- [ ] Design team 
- [ ] Development team 

### Past Month Accomplishments
- [ ] Product Team
- [ ] UXR team
- [ ] Content team
- [ ] Design team 
- [ ] Development team 

### Our Overall Direction

### What's Next Month / What we need from each other / Dependencies (by team)

**Comment by mariaweissman** [comment]
# 2025-03-06 Agenda and Notes

### Meeting Facilitator:
### Meeting Notetaker:

### Roll Call
- [ ] Product Team
- [ ] Research team
- [ ] Content team
- [ ] Design team 
- [ ] Development team 

### Past Month Accomplishments
- [ ] Product Team
- [ ] UXR team
- [ ] Content team
- [ ] Design team 
- [ ] Development team 

### Our Overall Direction

### What's Next Month / What we need from each other / Dependencies (by team)
- #1664 recap?

**Comment by mariaweissman** [comment]
# 2025-03-03 Agenda and Notes / Async

### Meeting Facilitator:
### Meeting Notetaker:

### Roll Call
- [ ] Product Team
- [ ] Design team 
- [ ] Development team 
- [ ] Research team - TDM
- [ ] Content team - TDM/ April off

### Items to Discuss

- [ ] Maria meeting with Bonnie Friday [re credits page proposal](https://github.com/hackforla/expunge-assist/issues/379#issuecomment-2767290988)
- [ ] April items:
        - [ ] Design: Continue working on Stakeholder Review issues as able (#1706)
               - [ ] Simultaneously, can start working on design system/ UI kit?
               - [ ] Note: #1699 hasn't had an update in about a month and needs one
        - [ ] Dev: 


### What's Next Month / What we need from each other / Dependencies (by team)

**Comment by mariaweissman** [comment]
# 2025-05-01 Agenda and Notes

### Meeting Facilitator:
### Meeting Notetaker:

### Roll Call
- [ ] Product Team
- [ ] Research team
- [ ] Content team
- [ ] Design team 
- [ ] Development team 

### Past Month Accomplishments
- [ ] Product Team
- [ ] UXR team
- [ ] Content team
- [ ] Design team 
- [ ] Development team 

### Our Overall Direction & Items to Discuss
- [ ] **Recruiting**: Let's Brainstorm, Collaborate & Clarify
    - What's needed? What's been done? What works well? What could be improved?
- [ ] **Stakeholder 2.0**: Progress and blockers updates #1461
- [ ] **Next Scheduled Org Meetings**: 
    - May 2: Should it be postponed for June, or is there anything you'd like team to review with Bonnie
    - June 6: FYI Maria out of town this day and unavailable
- [ ] **Anything else?**: 

### What's Next Month / What we need from each other / Dependencies (by team)

**Comment by raeraeicky** [comment]
# [2025-11-06] Agenda and Notes

### Meeting Facilitator: Rachel
### Meeting Notetaker:

### Roll Call
- [ ] Product Team
- [ ] Research team
- [ ] Content team
- [ ] Design team 
- [ ] Development team 

### Past Month Accomplishments
- [ ] Product Team
- [ ] UXR team
- [ ] Content team
- [ ] Design team 
- [ ] Development team 

### Our Overall Direction
- Lots of team members leaving, some things to think about:
      - Merge Content back into UXR or UXD?
      - Ask Bonnie for resources?
 - Monthly Bonnie PM meeting tomorrow: other leads welcome if they want to join, what to discuss

### What's Next Month / What we need from each other / Dependencies (by team)

**Comment by shraddhasaraf** [comment]
# [2026-02-05] Agenda and Notes

### Meeting Facilitator: Shraddha
### Meeting Notetaker: Namita's Read.ai

### Roll Call
- [x] Product Team
- [x] Research team
- [ ] Content team
- [x] Design team 
- [ ] Development team 

### Past Month Accomplishments
- [ ] Product Team
- [ ] UXR team
- [ ] Content team
- [ ] Design team 
- [ ] Development team 

### Our Overall Direction
- Content is merged with Design but we dont have any content writers. We can onboard new people when we have some work for content and then decide if we still want content as part of Design.
- 

### What's Next Month / What we need from each other / Dependencies (by team)
